### PR TITLE
#45, add registration page for pacient

### PR DESCRIPTION
**Pull request by MaksMatch**

### CODE REVIEW

File: src/pages/pacient/login/LoginPacient.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import LoginForm from "../../../Forms/LoginForm";
+import LoginForm from "../../../forms/LoginForm";
```
Comment (kar-now) [inline src/pages/pacient/login/LoginPacient.js:1]: why is it changed?

Comment (kar-now) [inline src/pages/pacient/login/LoginPacient.js:1]: "Forms" is the correct folder( not "forms")

File: src/pages/pacient/registration/RegistrationPacient.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import RegistrationPacientForm from "../../../forms/RegistrationPacientForm";
```
Comment (kar-now) [inline src/pages/pacient/registration/RegistrationPacient.js:1]: change "forms" - > "Forms"

File: src/pages/doctor/login/LoginDoctor.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import LoginForm from "../../../Forms/LoginForm";
```
Comment (kar-now) [inline src/pages/doctor/login/LoginDoctor.js:1]: change "forms" - > "Forms"

**Review by kar-now — CHANGES_REQUESTED** [review]
wrong forms imports (folder is calles "Forms" and not "forms")

**Comment by MaksMatch** [comment]
fixed

**Review by kar-now — APPROVED** [review]

**Pull request merged**